### PR TITLE
Added tests execution and modules verification

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Added tests execution and modules verification

### What issues does this PR fix or reference?
n/a

### CODE REVIEW

File: build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -11,13 +11,18 @@
 #
 
 FROM golang:1.13-alpine3.12 as builder
+ARG CGO_ENABLED=0
```
Comment (sparkoo) [inline build/dockerfiles/Dockerfile:14]: how are these used?

File: build/dockerfiles/Dockerfile [hunk 2]
```diff
@@ -11,13 +11,18 @@
...
+ARG GOOS=linux
 RUN apk add --no-cache ca-certificates
 RUN adduser -D -g '' appuser
 WORKDIR /go/src/github.com/che-incubator/configbump
-COPY . /go/src/github.com/che-incubator/configbump
+# copy go.mo go.sum
```
Comment (sparkoo) [inline build/dockerfiles/Dockerfile:19]:
```suggestion
# copy go.mod go.sum
```

File: build/dockerfiles/rhel.Dockerfile [hunk 3]
```diff
@@ -20,9 +22,14 @@ ENV PATH=/opt/rh/go-toolset-1.13/root/usr/bin:$PATH
 ENV GOPATH=/go/
 USER root
 WORKDIR /go/src/github.com/che-incubator/configbump
+# copy go.mo go.sum
```
Comment (sparkoo) [inline build/dockerfiles/rhel.Dockerfile:25]:
```suggestion
# copy go.mod go.sum
```

File: build/dockerfiles/Dockerfile (hunk 1 above)
Comment (skabashnyuk) [reply build/dockerfiles/Dockerfile:14]: used by go test and go build tools. it is equivalent to `ENV`, **potentially**  used for the mutli-arch build.  Can be replaced with `ENV`

Comment (sparkoo) [reply build/dockerfiles/Dockerfile:14]: I don't see it used there. I would assume something like `CGO_ENABLED=$CGO_ENABLED GOOS=$GOOS` on test and build lines.

Comment (skabashnyuk) [reply build/dockerfiles/Dockerfile:14]: AFAIK it's environment variable. I assume it's not necessary. @metlos wdyt?

Comment (sparkoo) [reply build/dockerfiles/Dockerfile:14]: ah ok, make sense

Comment (skabashnyuk) [reply build/dockerfiles/Dockerfile]: replaced with ENV

**Pull request merged**